packages/styles/accordion.css

```diff
@@ -3,67 +3,74 @@
   --accordion-trigger-background-color-expanded: var(--white);
   --accordion-trigger-text-color: var(--gray-90);
   --accordion-trigger-text-color-hover: var(--accent-primary);
-  --accordion-trigger-box-shadow-hover: var(--accent-primary);
+  --accordion-trigger-box-shadow-expanded: var(--accent-primary);
   --accordion-trigger-border-color: var(--gray-40);
   --accordion-trigger-icon-color: var(--gray-90);
   --accordion-panel-background-color: var(--white);
-  --accordion-panel-text-color: var(--gray-90);
+  --accordion-panel-text-color: var(--gray-80);
   --accordion-panel-border-color: var(--gray-40);
   --accordion-box-shadow-color: rgba(0, 0, 0, 0.15);
 }
 
 .cauldron--theme-dark {
   --accordion-trigger-background-color: var(--accent-medium);
-  --accordion-trigger-background-color-expanded: var(--accent-dark);
-  --accordion-trigger-text-color: var(--white);
-  --accordion-trigger-text-color-hover: var(--accent-light);
-  --accordion-trigger-box-shadow-hover: var(--accent-info);
+  --accordion-trigger-background-color-expanded: var(--accent-medium);
+  --accordion-trigger-text-color: var(--accent-light);
+  --accordion-trigger-text-color-hover: var(--white);
+  --accordion-trigger-box-shadow-expanded: var(--accent-info);
   --accordion-trigger-border-color: var(--stroke-dark);
   --accordion-trigger-icon-color: var(--white);
   --accordion-panel-background-color: var(--accent-medium);
-  --accordion-panel-text-color: var(--white);
+  --accordion-panel-text-color: var(--accent-light);
   --accordion-panel-border-color: var(--stroke-dark);
 }
 
+.Accordion:has(.Accordion__trigger[aria-expanded='true']) {
+  filter: drop-shadow(0px 2px 2px rgba(0, 0, 0, 0.15));
```

Comment:
Why `filter: drop-shadow`?

Reply:
This should be using `box-shadow` and not `filter`. `box-shadow` is more widely supported, and the css properties aren't always a 1:1. I'm guessing this should be:

```css
box-shadow: 0 2px 2px rgba(0, 0, 0, 0.15);
```